docs/developers/maintenance.md

```diff
@@ -0,0 +1,61 @@
+# Maintenance
+
+This document describes the maintenance tasks that need to be performed from time to time. 
+To rare to depend only on personal memory. 
```

Comment:
```suggestion
This document describes maintenance tasks that need to be performed from time to time. Its purpose is to avoid relying on personal memory alone, and it should be updated regularly.
```